addons/project_issue/models/project_project.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    @api.model
+    def _get_alias_models(self):
+        return [('project.task', "Tasks"), ("project.issue", "Issues")]
+
+    issue_count = fields.Integer(compute='_compute_issue_count', string="Issues")
+    issue_ids = fields.One2many('project.issue', 'project_id', string="Issues", domain=[('stage_id.fold', '=', False)])
+    label_issues = fields.Char('Use Issues as', help="Customize the issues label, for example to call them cases.", default='Issues')
+    use_issues = fields.Boolean(default=True)
+
+    @api.multi
+    def _compute_issue_count(self):
+        issues_data = self.env['project.issue'].read_group([('project_id', 'in', self.ids), ('stage_id.fold', '=', False)], ['project_id'], ['project_id'])
+        result = dict((data['project_id'][0], data['project_id_count']) for data in issues_data)
+        for project in self:
+            project.issue_count = result.get(project.id, 0)
+
+    @api.onchange('use_issues', 'use_tasks')
+    def on_change_use_tasks_or_issues(self):
+        if self.use_tasks and not self.use_issues:
+            self.alias_model = 'project.task'
+        elif not self.use_tasks and self.use_issues:
+            self.alias_model = 'project.issue'
+
+    @api.model
+    def create(self, vals):
+        self._check_create_write_values(vals)
```

Comment:
no need to define this method. onchange already did this. so no need to overrride create and write method.